src/Admin/OnboardingWizardSubmenu.php

```diff
@@ -40,7 +49,7 @@ public static function get_registration_action() {
 	 */
 	public function register() {
 		add_submenu_page(
-			AMP_Options_Manager::OPTION_NAME,
+			'',
 			__( 'Onboarding Wizard', 'amp' ),
```

Comment:
This would be the page title, so we should add something to make it unique.

```suggestion
			__( 'AMP Onboarding Wizard', 'amp' ),
```

Reply:
I believe this is now obsolete because the `admin_title` filter is being used.

Reply:
But it doesn't hurt.

Reply:
Also because it reduces the number of strings to translate. Done in 417e82415.